cmd/config/go.mod

```diff
@@ -23,4 +27,7 @@ require (
 //
 // releasing/releasemodule.sh will remove this line and set the require version
 // to the kyaml version specified in releasing/VERSIONS
-replace sigs.k8s.io/kustomize/kyaml v0.0.0 => ../../kyaml
+replace (
+	sigs.k8s.io/kustomize/kyaml v0.0.0 => ../../kyaml
+	sigs.k8s.io/kustomize/kyaml v0.1.13 => ../../kyaml
```

Comment:
please add a comment as to why both are needed
also, the Go-based releasing program is looking good; i'd like to delete the bash script and the VERSIONS file asap.

can you try the go program next time ?
try this:
```
(cd releasing/releasing; go run . list)
(cd releasing/releasing; go run . release kyaml minor)
```

haven't added a make target for it yet

Reply:
SGTM

Reply:
Thanks for the link to the new release tooling.  So glad to see it.